packages/theme-chalk/src/input.scss

```diff
@@ -179,13 +179,6 @@
     pointer-events: none;
   }
 
-  @include when(active) {
```

Comment:
Why are these code useless?

Reply:
because there is no way to change the status to active